examples/platform/silabs/matter_config.h

```diff
@@ -22,6 +22,9 @@
 #include <platform/CHIPDeviceLayer.h>
 #include <platform/KeyValueStoreManager.h>
 
+// TODO define
```

Comment:
this comment does not help reading code. Consider removing or explaining in some other way (maybe in the ICDManager header itself.

Reply:
Shouldn't be there. Reminder for me to surround the include with the associated define.

Reply:
isn't it already available
`CHIP_DEVICE_CONFIG_ICD_SERVER_ENABLE`